.github/team.json

```diff
@@ -3,6 +3,7 @@
     {
       "name": "Querying Components",
       "label": ".Team/QueryingComponents",
+      "projectUrl": "https://github.com/orgs/metabase/projects/74",
```

Comment:
Hey @metabase/querying-components, would you like PRs you create get assigned to the board?